source/localizable/tutorial/service.md

```diff
@@ -0,0 +1,293 @@
+For Super Rentals, we want to add the ability to display maps of where each rental is located.  To implement this feature, we will take advantage of several Ember concepts:
+
+1. A component to display a map on each rental listing.
+2. A service to keep a cache of rendered maps to use in different places in the application.
+3. A utility function to create a map from the Google map API.
+
+We'll work on this feature outside-in, meaning we'll start with the display of the map and work our way back to the actual access of the Google Maps API.
+
+
+### Display Maps With a Component
+
+For our maps component, all we'll require is a simple component interface that we can simply drop into a template.
+The component will render a map pinned to whatever location we provide it.
+Lets start by appending the component as we want it to the end of the `rental-listing` template, and provide it with the city of the current rental.
```

Comment:
Instead of the above paragraphs, I went to edit things a bit and simplified down to:

> We'll start by adding a map component that shows the rental's city on a map